Library/Homebrew/cmd/update

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh # brew-update.sh
+[ -n "$HOMEBREW_VERBOSE" ] && set -x
```

Comment:
This (that is, `set -x` being woefully deficient for logging/debugging, particularly with the backgrounded jobs) was in large part the reason I put together the shell utility library. You can look forward to something much prettier.